lib/common/client/base.rb

```diff
@@ -67,6 +68,28 @@ def request(method, path, params = {}, headers = {})
         raise client_error
       end
 
+      def sanitize_headers!(method, path, params, headers)
+        unmodified_headers = headers.dup
+        headers.transform_keys!(&:to_s)
+
+        headers.transform_values! do |value|
+          if value.nil?
+            unless Rails.env.test?
+              log_message_to_sentry(
+                'nil headers bug',
+                :info,
+                unmodified_headers: unmodified_headers, method: method, path: path, params: params, client: inspect,
```

Comment:
could get some double logging here as it logs all the headers for each one that's nil

Reply:
Sorry, not following you.  I removed the sentry logging from the `request` method itself, and moved it into this `sanitize` method.  So we should only get logging once per request, and only for the `nil` case.

Or perhaps I am missing something (which is likely the case)?